paper/paper.md

```diff
@@ -48,7 +48,7 @@ By 2019 it was clear that re-designing HyP3 to take advantage of evolving techno
 The first redesigned on-demand workflow made publicly available was Radiometric Terrain Correction (RTC) in [month year], followed by InSAR in [month year]. Since that time, new image processing services have been added to HyP3 including a glacier velocity tracking tool, and flood inundation mapping workflows. The user base grew to XX by [date], with users producing XX image products on average every month. Since 2021, a range of users have recognized the utility of HyP3. HyP3 was deployed within NASA's Earthdata Cloud in 2022, and many other deployments of HyP3 now exist at other institutions (see the Ongoing Projects section for details).
 
 ## HyP3 Architecture
-HyP3 is a cloud processing pipeline that is designed with a serverless architecture and to follow AWS architecture best practices. It also strives to make a clear separation between the plugins that perform the image processing and the pipeline in which this task is performed. See Figure FF for the AWS architecture diagram.
+HyP3 is a cloud processing pipeline that is designed with a serverless architecture and to follow AWS Well-Architectured Framework. It also strives to make a clear separation between the plugins that perform the image processing and the pipeline in which this task is performed. See Figure FF for the AWS architecture diagram.
```

Comment:
AWS has a "Well-Architectured Framework'' [outline](https://docs.aws.amazon.com/wellarchitected/latest/framework/welcome.html) that outlines its best practices. I was a bit confused as to what exactly this means. Do we want to state it is a `operating secure, reliable, efficient, cost-effective, and sustainable workload in the AWS Cloud`?